src/editor/Collaboration.js

```diff
@@ -86,13 +91,18 @@ define(function (require, exports, module) {
             case "file-rename":
                 oldPath = Path.join(rootDir, payload.oldPath);
                 newPath = Path.join(rootDir, payload.newPath); 
+                _renaming[oldPath] = true;
+                CommandManager.execute("bramble.renameFile", {from: oldPath, to: payload.newPath})
```

Comment:
`newPath` and `payload.newPath` differ since `newPath` is absolute and `payload.newPath` is relative, and the `bramble.renameFile` function expects a relative path as the second parameter.